pkg/scheduler/internal/cache/interface.go

```diff
@@ -56,6 +56,9 @@ import (
 // - Both "Expired" and "Deleted" are valid end states. In case of some problems, e.g. network issue,
 //   a pod might have changed its state (e.g. added and deleted) without delivering notification to the cache.
 type Cache interface {
+	// NodeCount returns the number of nodes in the cache.
```

Comment:
Also say here to not use outside of tests.

Reply:
done.